Jenkinsfile

```diff
@@ -147,9 +155,10 @@ pipeline {
 
                 // Build our ISO
                 dir('stacki-iso-builder') {
-                    retry(1) {
-                        // Give the build up to 60 minutes to finish
-                        timeout(60) {
+                    // Retry a few times, because CentOS mirrors are flaky
+                    retry(2) {
+                        // Give the build up to 120 minutes to finish
+                        timeout(120) {
```

Comment:
Does this here also need the try/catch fix? I see retry(2) right before this timeout.

Reply:
I think it will be fine without one, because the `sh` step should fail. The timeout probably isn't actually needed, because I'm never seen it actually get stuck anywhere.

If we end up getting aborts from timeouts before Jenkins fixes their bug, I'll add a try/catch. Adding in the script try/catch stuff is ugly, so I want to avoid it where I can.